internal/pbio/pbio.go

```diff
@@ -0,0 +1,142 @@
+// Protocol Buffers for Go with Gadgets
```

Comment:
Content of this file is (partially) copy-pasted from gogo/test/thetest.pb.go

I copied && adapted ONLY varint implementation that is used for feedx. I don't think we'll ever need other encoding for it - and, IIRC, we implement only varint encoding in pbio gem.